.github/workflows/gradle-build-and-test.yml

```diff
@@ -23,6 +23,14 @@ jobs:
         run: ./gradlew jacocoTestReport --info
         working-directory: TrafficCapture
 
+      - name: Upload Test Results
+        uses: actions/upload-artifact@v3
+        with:
+          name: test-results
+          path: |
+            TrafficCapture/trafficReplayer/build/test-results/slowTest
+            TrafficCapture/trafficReplayer/build/reports/tests/slowTest
+
```

Comment:
Does this only upload slow test logs? It would be nice to have all test logs here